src/Matrix.cpp

```diff
@@ -309,6 +381,21 @@ int* Matrix::getJA() {
 
 
 /**
+ * @brief Get the JA component (excluding the diagonal) of the CSR form of the
```

Comment:
Perhaps it might be nice to include a `@details` comment block describing what the `JA` component physically represents?